components/src/components/container/component-checkbox-selector.ts

```diff
@@ -0,0 +1,75 @@
+import { customElement, property, state } from 'lit/decorators.js';
+import { html } from 'lit';
+import { TailwindElement } from '../../tailwind-element';
+
+type CheckboxItem = {
+    label: string;
+    checked: boolean;
+};
+
+@customElement('gs-component-checkbox-selector')
+export class ComponentCheckboxSelector extends TailwindElement() {
+    @property({ type: Array })
+    items: CheckboxItem[] = [];
+
+    @property({ type: String })
+    label = 'label';
+
+    @property({ type: String })
+    size = 'xs';
+
+    @state()
+    private checkedItems: CheckboxItem[] = [];
+
+    override updated(changedProperties: Map<string | number | symbol, unknown>) {
+        if (changedProperties.has('items')) {
+            this.checkedItems = this.items.map((item) => {
+                return item;
+            });
+        }
+    }
+
+    flipCheckedItem(index: number) {
+        this.checkedItems = this.checkedItems.map((checkedItem, checkedIndex) => {
+            if (checkedIndex === index) {
+                return {
+                    label: checkedItem.label,
+                    checked: !checkedItem.checked,
+                };
+            }
+            return checkedItem;
+        });
+    }
+
+    override render() {
+        return html`
+            <div class="dropdown">
+                <div tabindex="0" role="button" class="btn btn-${this.size}">${this.label}</div>
```

Comment:
```suggestion
                <div tabindex="0" role="button" class="btn ${this.size}">${this.label}</div>
```
I quite sure that optimization for only actually used css classes doesn't work, if we use them dynamically. Is the `size` prop actually used currently? We can also remove it for now and check later what customization abilities we need here.

Reply:
It is not used. I added it, because we might want to change the size of the text somehow later on. We have many different sizes right now. I removed it here and we can add it later on.